### ISSUE TITLE
Average fewer than 100 mouseclicks over working week. 

### ISSUE THREAD

**Issue by joereddington** [joereddington/Projects-Public#30]
# Motivation
I work a lot better when I use only the keyboard - I 'check' sites less, and the places I do use are much more useful. Also I get a lot of pleasure out of knowing all the keyboard shortcuts and feeling like a proper computer expert. 

So, I'm using [Whatpulse](https://whatpulse.org/) to count mouse clicks (and various other bits) 

# Articles
* [https://gist.github.com/lornajane/3892c39098cf70baa9c7a1874cddf233](https://gist.github.com/lornajane/3892c39098cf70baa9c7a1874cddf233)
* [https://webmasters.stackexchange.com/q/131325/14932](https://webmasters.stackexchange.com/q/131325/14932}

# Tools 
* Whatpulse 
* Jumpcut (already vital) 
* Spectucal (already vitial) 
* 'mouseless' firefox add-on
* Shortcat (osx only)  ('mouseless' but outside of the browser)
* "Putting the mouse in a box"
* Making a big list of all the useful shortcuts 
* Shortcat (osx only) 
* https://defkey.com/lastpass-shortcuts looks pretty good 

# Plan 
* Keep trying and googling when things go wrong 
* Keep a log of the things that turn out to be hard 
* Review the whatpulse results regularly.

# Questions: 
* Can I have a visual indicator of where tab would go next? 
* Excel online is *hard* 

### Log 
* Needed mouse to switch wifi off and on.. 
* Needed mouse for excel online 
* Needed to work Francesca's blog 
* Creating a pdf from print 
* accepting the shortcat licence aggreeemnt! 
* paying on dreamhost for some reason. 

# Shortcuts and URLS 
 
CMD-SHIFT-? (CMD-? really) is 'search the menu bar' 
"you can trigger your menu bar icons by pressing Fn+Control+F8 on your keyboard. The left-most menu bar icon will be highlighted" 

## Quickbooks 

New bill         - https://c14.qbo.intuit.com/app/bill

## Shortcat 
Cmd-Shift-space        - Open shortcat window

## Google Drive/Sheets
<Ctrl><Alt><f>         -        Open file menu (shortcut keys underlined from there) 

## Gmail
Open in a new window | -	Focus main window
<Esc>                  -	Focus latest chat or compose
<Ctrl>+.               -	Advance to next chat or compose
<Ctrl>+,               -	Advance to previous chat or compose
<⌘>+<Enter>            -	Send
<⌘>+<Shift>+c          -	Add Cc recipients
<⌘>+<Shift>+b          -	Add Bcc recipients
<⌘>+<Shift>+f          -	Access custom from
<⌘>+<Shift>+d          -	Discard draft
<⌘>+k                  -	Insert a link
<⌘>+;                  -	Go to previous misspelled word
<⌘>+'                  -	Go to next misspelled word
<⌘>+<Shift>+'          -	Open spelling suggestions
<⌘>+<⌥>+.              -	Go to next section
<⌘>+<⌥>+,              -	Go to previous section
<⌘>+z                  -	Undo
<⌘>+y                  -	Redo
<⌘>+<Shift>+           -	Decrease text size
<⌘>+<Shift>++          -	Increase text size
<⌘>+b                  -	Bold
<⌘>+i                  -	Italics
<⌘>+u                  -	Underline
<⌘>+<Shift>+5          -	Previous font
<⌘>+<Shift>+6          -	Next font
<⌘>+<Shift>+7          -	Numbered list
<⌘>+<Shift>+8          -	Bulleted list
<⌘>+<Shift>+9          -	Quote
<⌘>+[                  -	Indent less
<⌘>+]                  -	Indent more
<⌘>+<Shift>+l          -	Align left
<⌘>+<Shift>+e          -	Align center
<⌘>+<Shift>+r          -	Align right
<⌘>+<Shift>+,          -	Set right-to-left
<⌘>+<Shift>+.          -	Set left-to-right
<⌘>+<Shift>+x          -	Strikethrough
<⌘>+\                  -	

## Mouseless (Firefox Add-on) 
h                      - Show "blobs" with characters
Enter and Shift+Enter  - Click blob or open in new tab
Ctrl+Enter             - Save the URL of the selected link to the clipboard
k and l                - Scroll up and down
_ and +                - Scroll up and down fast
Ctrl+j and Ctrl+;      - Go back/forward in history
j and ;                - Switch to the tab to the left/right of the current one
J and :                - Move the tab left/right

**Comment by joereddington** [comment]
I have switched on MouseKeys, after some dififcultity.

**Comment by joereddington** [comment]
Update 
This is working well, except for facebook and darktable.  I've done some more looking up of facebook shortcuts, so that should improve later.

**Comment by joereddington** [comment]
Set up whatpulse on public today.

**Comment by joereddington** [comment]
Darktable really was the one for this. Recently my VIM has improved a fair bit so many this is room for improvement here.

**Comment by joereddington** [comment]
The next activity is to put Whatpulse on for a day and see what happens.

**Comment by joereddington** [comment]
Suspended during occupation.

**Comment by joereddington** [comment]
Worth writing up 

* I wanted to learn some faster ways of using the keyboard, because it feels more skillful even it isn't 
* I used whatpulse 
* Here are my results for a day.
* I used this extension, which I really like, and this shortcut was invaluable. I ended up switching from Excel to google sheets
* Photo editing using Darktable is near impossible, for No Good Reason. 
* some websites make it really easy, some make it hard.
